internal/page/fixtures/voucher.go

```diff
@@ -32,6 +33,7 @@ func Voucher(
 	progressValues := []string{
 		"confirmYourName",
 		"verifyDonorDetails",
+		"confirmYourIdentity",
```

Comment:
Do we want to add this to the fixtures page too?

Reply:
I did and I forgot, I'll add it in the next one